app.js

```diff
@@ -0,0 +1,262 @@
+'http://www.seriouseats.com/recipes/2015/03/gold-rush-bourbon-honey-cocktail-drink-recipe.html',
+'http://www.seriouseats.com/recipes/2012/02/drinking-in-season-grapefruit-ginger-sparkling-wine-cocktail-recipe.html',
+'http://www.seriouseats.com/recipes/2014/03/simple-cocktail-blushing-betty-bourbon-grapefruit-easy-drink-recipe.html',
+'http://www.seriouseats.com/recipes/2008/05/daiquiri-recipe.html',
+'https://www.popsugar.com/food/Hot-Cold-Butterbeer-Recipe-380247',
+'https://www.popsugar.com/food/Pomegranate-Champagne-Cocktail-26435241',
+'http://youngaustinian.com/2014/02/13/modern-austen-blood-orange-rickey/',
+'https://www.popsugar.com/food/Salty-Dog-Cocktail-Recipe-2674478',
+'https://www.popsugar.com/food/Coffee-Martini-37944195']
+
+
+var cocktailRecipeBox = [];
+console.log(cocktailRecipeBox);
+
+
+// Constructor Function to create Cocktail objects
+function Cocktail(name, imagePath, ingredient1, ingredient2, ingredient3, link){
+  this.name = name;
+  this.imagePath = imagePath;
+  this.ingredient1 = ingredient1;
+  this.ingredient2 = ingredient2;
+  this.ingredient3 = ingredient3;
+  this.link = link;
+  cocktailRecipeBox.push(this);
+}
+// This IIFE Creates the cocktail objects on load
+(function() {
+
+  for (var i in cocktailNames){
+    var newCocktail = new Cocktail(cocktailNames[i], 'assets/assetspng/' + cocktailNames[i] + '.png', ingredient1Array[i], ingredient2Array[i],
+    ingredient3Array[i], linkArray[i])
+  };
+})()
+
+var Tracker = {
+  userSelectedIngredientsArray: [],
+  cocktailOptionsRound1: [],
+  cocktailOptionsRound2: [],
+  cocktailOptionsRound3: [],
+
+  dynamicallyLoadedIngredients: function(){
+    var getIngredientOne = document.getElementById('ing1');
+    var getIngredientTwo = document.getElementById('ing2');
+    var getIngredientThree = document.getElementById('ing3');
+
+    for (var i in ingredient1Array) {
+      var createOptionsOne = document.createElement('option');
+      createOptionsOne.value = ingredient1Array[i];
+      createOptionsOne.textContent = ingredient1Array[i];
+      getIngredientOne.appendChild(createOptionsOne);
+      console.log(createOptionsOne);
+    };
+
+    for (var i in ingredient2Array) {
+      var createOptionsTwo = document.createElement('option');
+      createOptionsTwo.value = ingredient2Array[i];
+      createOptionsTwo.textContent = ingredient2Array[i];
+      getIngredientTwo.appendChild(createOptionsTwo);
+    };
+
+    for (var i in ingredient3Array) {
+      var createOptionsThree = document.createElement('option');
+      createOptionsThree.value = ingredient3Array[i];
+      createOptionsThree.textContent = ingredient3Array[i];
+      getIngredientThree.appendChild(createOptionsThree);
+    };
+  },
+
+
+  // ingredients that the user select get push into an array to be used in the verification process.
+
+  handleUserFormSubmit: function(event) {
+    Tracker.userSelectedIngredientsArray = [];
```

Comment:
Similar comment as above. There's repetitive code here that could be separated out into a helper function and given arguments.